### ISSUE TITLE
Investigate use of Bloodhound, etc.

### ISSUE THREAD

**Issue by qqmyers** [gdcc/dataverse-external-vocab-support#16]
The ROR community has some type-ahead demos that use Bloodhound.  See https://github.com/ror-community/ror-typeahead-demos .  It should be possible to adapt our existing script to use Bloodhound and/or other aspects of the demos' designs if it looks like they have advantages (or just to reduce the amount of custom code we have).

**Comment by pdurbin** [comment]
Yes, seems nice. It seems to be demo-able at https://ror-community.github.io/ror-typeahead-demos/

![Screen Shot 2023-05-04 at 2 15 06 PM](https://user-images.githubusercontent.com/21006/236292976-3dabc1f2-2530-4f85-a6d3-f45a4246419c.png)
